apps/front-end/front-office/src/app/reducers/game/game.reducer.spec.ts

```diff
@@ -50,8 +55,8 @@ describe('Game Reducer', () => {
 
       expect(result).toEqual({
         ...initialGameState,
-        score: 1,
-        times: [1000],
+        score: 0,
+        times: [], //FIXME
```

Comment:
times not working?

Reply:
FAIL apps/front-end/front-office/src/app/reducers/game/game.reducer.spec.ts (6.253 s)
  ● Game Reducer › game › correctAnswer should set score

    expect(received).toEqual(expected) // deep equality

    - Expected  - 4
    + Received  + 2

    @@ -6,11 +6,9 @@
        "loaded": false,
        "player": null,
        "quiz": null,
        "remainingQuestions": Array [],
        "remainingTries": 0,
    -   "score": 1,
    -   "times": Array [
    -     1000,
    -   ],
    +   "score": 0,
    +   "times": Array [],
        "wrongQuestions": Array [],
      }

      54 |       const result = gameReducer(initialGameState, action);
      55 |
    > 56 |       expect(result).toEqual({
         |                      ^
      57 |         ...initialGameState,
      58 |         score: 1,
      59 |         times: [1000],

      at src/app/reducers/game/game.reducer.spec.ts:56:22
      at _ZoneDelegate.Object.<anonymous>._ZoneDelegate.invoke (../../../node_modules/zone.js/bundles/zone-testing-bundle.umd.js:412:30)
      at ProxyZoneSpec.Object.<anonymous>.ProxyZoneSpec.onInvoke (../../../node_modules/zone.js/bundles/zone-testing-bundle.umd.js:3833:43)
      at _ZoneDelegate.Object.<anonymous>._ZoneDelegate.invoke (../../../node_modules/zone.js/bundles/zone-testing-bundle.umd.js:411:56)
      at Zone.Object.<anonymous>.Zone.run (../../../node_modules/zone.js/bundles/zone-testing-bundle.umd.js:169:47)
      at Object.wrappedFunc (../../../node_modules/zone.js/bundles/zone-testing-bundle.umd.js:4333:34)